server/templates/stations.html

```diff
@@ -18,8 +18,8 @@ <h3>Ground stations</h3>
             <td><a href="/station/{{ s.station_id }}">{{ s.name }}</a></td>
             <td>{{ s.coords }}</td>
             <td>{{ s.cnt }} [<a href="/obslist?station_id={{s.station_id}}">list</a>]</td>
-            <td>{{ s.registered }}</td>
-            <td>{{ s.lastobs }}</td>
+            <td>{{ s.registered.isoformat(" ", "seconds") }}</td>
```

Comment:
You put the same datetime format in several places. Maybe you want to use [template filter](https://stackoverflow.com/a/4830620) for define format in one place. It isn't important for me and you may to ignore this note. It is here only if you want to learn something new.

Reply:
Cool trick. Added template filter.